test/ranges/subset.js

```diff
@@ -0,0 +1,48 @@
+const t = require('tap')
+const subset = require('../../ranges/subset.js')
+
+// sub, dom, expect, [options]
+const cases = [
+  ['1.2.3', '1.2.3', true],
+  ['1.2.3 1.2.4', '1.2.3', true],
```

Comment:
this seems off - the first one allows 1.2.4, but the second one doesn’t, so shouldn’t it be false?

Reply:
The first one is a null set.  It's `1.2.3` AND `1.2.4`, not `1.2.3 || 1.2.4`.

Reply:
gotcha